introduction/java_loops_II

```diff
@@ -0,0 +1,34 @@
+import java.lang.Math;
+import java.util.Scanner;
+import java.lang.Object;
+
+class Solution {
+  static private final int EXPONENT = 2;
+  private static String loops(int firstNumber, int secondNumber, int power) {
```

Comment:
import java.lang.StringBuilder;
    
    private static String generateSeries(int coefficientA, int coefficientB, int seriesLength) {
      int bMultiplier = 1;
      int seriesTerm = coefficientA;
      StringBuilder resultBuilder = new StringBuilder();
      for (int i = 0; i < seriesLength; i++) {
        seriesTerm += coefficientB * bMultiplier;
        resultBuilder.append(seriesTerm + " ");
        bMultiplier = bMultiplier * B_MULTIPLIER_BASE;
      }
      return resultBuilder.build();
    }